docs/pages/admin-guides/access-controls/rbac/resource-rbac.mdx

```diff
@@ -0,0 +1,108 @@
+
+---
+title: Getting Started With Per-Resource Role-Based Access Control
+description: Implementing fine-grained access control with RBAC.
+---
+
+
+## What is per-resource RBAC?
+
+This granular approach allows you to grant more precise management of user permissions.
+
+Unlike broader, traditional role-based permissions, which can apply to all resources within a namespace or across an entire infrastructure, per-resource RBAC enables administrators to control user access on a per-resource basis. 
+This feature is especially valuable in complex environments with diverse user groups, where some users need restricted access to sensitive systems while others may require broad access.
```

Comment:
This sounds a bit like marketing language to me, rather than documentation. Is there a way to explain the feature for a reader who is already committed to using Teleport and wants to set up RBAC?

Reply:
Rephrased